java/ql/src/Security/CWE/CWE-200/AndroidWebViewSettingsFileAccess.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Android WebSettings file access
+ * @kind problem
+ * @description Enabling access to the file system in a WebView can enable access to sensitive information.
```

Comment:
This sentence seems like it needs a little rewriting (repetition of "enable access"). We can wait for docs review though, if you prefer.